hw1/02-palindrome.js

```diff
@@ -1,10 +1,12 @@
 const elem = document.querySelector("input");
-
 elem.addEventListener("input", handleInput);
 
 function handleInput() {
   let rev = elem.value.split("").reverse().join("");
```

Comment:
Code looks good, but doesn't quite follow the airbnb style guide. Airbnb uses single quotes. I found this article helpful when setting up eslint to use the airbnb style: https://medium.com/@Tunmise/set-up-eslint-with-airbnb-style-guide-in-5-minutes-d7b4cc5707f8. Nice work!